main/keychain/src/main/java/org/cryptomator/keychain/LinuxSecretServiceAccess.java

```diff
@@ -0,0 +1,59 @@
+package org.cryptomator.keychain;
+
+import org.apache.commons.lang3.SystemUtils;
+import org.freedesktop.secret.simple.SimpleCollection;
+
+import javax.inject.Inject;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LinuxSecretServiceAccess implements KeychainAccessStrategy {
+	private SimpleCollection gnomeLoginKeyring = null;
```

Comment:
```suggestion
	private final Optional<SimpleCollection> gnomeLoginKeyring;
```

Reply:
Thanks for the suggestions. I will change the code accordingly right away.
And thank you also for the quick review!